### DISCUSSION TITLE
Easiest way to move my database to another acccount

### DISCUSSION THREAD

**Discussion by lofti198** [supabase/supabase#31325] [Questions]
I need to give my account to my client (but I used Google for authentication). What is the best way to do this? Just create new account (by my client) and somehow export/import there the database? Or is there any other way?

**Comment by GaryAustin1** [comment]
You should be able to do this by allowing your self access to be part of the organization team.  Then doing a transfer of project from org to org. 
https://supabase.com/docs/guides/platform/project-transfer

It says you do have to be owner in both orgs for the transfer.

Note: I've not tested this and just going by the docs.

**Reply by lofti198** [reply]
![image](https://github.com/user-attachments/assets/55006a84-8179-4085-9516-7502928ab1fe)
How do you think, am I supposed to change connection data when transferring?

**Reply by GaryAustin1** [reply]
Not sure what you are asking.  None of that changes if you just do an org to org transfer between to Supabase accounts.
